src/LoginResponseInterface.php

```diff
@@ -0,0 +1,17 @@
+<?php
+declare(strict_types=1);
+
+namespace Firehed\U2F;
+
+interface LoginResponseInterface
+{
+    public function getChallengeProvider(): ChallengeProvider;
+
+    public function getCounter(): int;
+
+    public function getKeyHandleBinary(): string;
+
+    public function getSignature(): string;
+
+    public function getSignedData(): string;
```

Comment:
This should, in principle, be identical across all formats, and liable to be adjusted or removed (this should always be `authData | sha256(clientData)`, where `authData` is `sha256(rpID) | userPresence | counter`)